stripe/checkoutCompleted.js

```diff
@@ -7,11 +7,23 @@ import { getEnvironment } from '../libs/utils-lib';
 // eslint-disable-next-line no-unused-vars
 import typings from '../typings/coupon';
 import { sendCouponEmail } from '../email/send-coupon';
+import * as secretManagerLib from '../libs/secretmanager-lib';
 
 const env = getEnvironment();
 export const main = async event => {
   const stripeSignature = event.headers['Stripe-Signature'];
-  const stripe = Stripe(env.STRIPE_API_SECRET_KEY);
+
+  const secretName = `${env.STAGE}/stripe`;
+  let stripeSecrets;
+  try {
+    console.log(`Get secret with name ${secretName}`);
```

Comment:
seems like the function `secretManagerLib.getSecrets` is better place for this `console.log`